src/shacl-shape-lib/connectors-shacl-shape.xsl

```diff
@@ -87,13 +217,16 @@
         <xsl:param name="connector"/>
         <xsl:variable name="sourceClassURI"
             select="f:buildURIfromLexicalQName($connector/source/model/@name)"/>
+        <xsl:variable name="normalisedSourceClassName"
+            select="replace($connector/source/model/@name, ':', '-')"/>
```

Comment:
why replace? input data shall never be altered. We had a task to remove any corrective behaviour.